.gitattributes

```diff
@@ -2,9 +2,12 @@
 # Auto detect text files and perform LF normalization
 
 *       text=auto
-*.cpp   -text
-*.h     -text
 
+# C++ files should use CRLF.
```

Comment:
Why do you need to force C++ files to be different than any other text file?  I have to go back and read up on the .gitattribute values as I cannot remember off of the top of my head, but basically Windows users probably have the config to do autocrlf, and I think `text=auto` honors that.

Reply:
Well I expanded the diff and I think the answer is in the comment below.

Reply:
git@github.com:JunielKatarn/react-native-windows.git

Reply:
Pretty much.
We shouldn't assume the developer has "canonical" settings just because they are using Windows.